### DISCUSSION TITLE
How to install chaos-mesh within a namespace scope

### DISCUSSION THREAD

**Discussion by legilimen** [chaos-mesh/chaos-mesh#2406] [Q&A]
Is it possible to install chaos-mesh with namespace scope and not cluster level scope. Currently when I install I get an error `cannot create resource "customresourcedefinitions" in API group "apiextensions.k8s.io" at the cluster scope`. I work in an env where i have access only to my namespace and we won't get cluster level access. I did find few discussions on this topic where ppl were suggesting some optional argument that can be passed to limit to namespace level access but am not able to find the right options.

**Comment by Fabianski-tr** [comment]
@legilimen Any update for this problem?
